github/repos_pages.go

```diff
@@ -62,6 +70,51 @@ func (s *RepositoriesService) EnablePages(ctx context.Context, owner, repo strin
 	}
 
 	return enable, resp, nil
+
+}
+
+type RemoveCName struct {
+	CName  *string `json:"cname"`
+	Source *string `json:"source,omitempty"`
+}
+
+func copyRemoveCName(opts *PageUpdate) *RemoveCName {
+	return &RemoveCName{
+		CName:  opts.CName,
+		Source: opts.Source,
+	}
+}
+
+// UpdatePages https://developer.github.com/v3/repos/pages/#update-information-about-a-pages-site
+func (s *RepositoriesService) UpdatePages(ctx context.Context, owner, repo string, opts *PageUpdate, removeCustomDomain bool) (*Response, error) {
+
```

Comment:
This function does not match Go style and does not match the style of the surrounding methods in this repo.
Please delete most of the blank lines and add separating blank lines to match the style of the surrounding methods.

If you would like me to explicitly tell you where blank lines should go, I will, but I'm betting you can figure it out by looking at other methods.